dashboard/app/views/programming_expressions/show.html.haml

```diff
@@ -1,3 +1,13 @@
 %script{src: webpack_asset_path('js/programming_expressions/show.js'),
   data: {programmingExpression: @programming_expression.summarize_for_show.to_json}}
 #show-container
+
+- if can? :edit, @programming_expression
+  = render layout: 'shared/extra_links' do
+    %strong= @programming_expression.name
+    %ul
+      - if Rails.application.config.levelbuilder_mode
+        %li= link_to 'Edit', edit_programming_expression_path(@programming_expression)
+      - else
+        %li= link_to 'Edit on levelbuilder', URI.join("https://levelbuilder-studio.code.org/", edit_programming_expression_path(@programming_expression)).to_s
```

Comment:
i like this pattern of linking to a levelbuilder edit page from studio! we could maybe use that for foorm stuff too

Reply:
I copied this almost directly from [the lesson show template](https://github.com/code-dot-org/code-dot-org/blob/b1ad326e5869b01cd1dff7013e13ac0bebdbea7c/dashboard/app/views/lessons/show.html.haml#L15), so I can't take credit for it. It only shows up for people with levelbuilder permissions on prod, which isn't a big group but is convenient in that case